applications/FluidDynamicsApplication/custom_processes/distance_smoothing_process.h

```diff
@@ -144,6 +322,8 @@ class KRATOS_API(FLUID_DYNAMICS_APPLICATION) DistanceSmoothingProcess : public P
     ModelPart& mrModelPart;
     Model& mrModel;
     std::string mAuxModelPartName;
+    bool mAllConditionsAsBoundary;
```

Comment:
What is the intended usage of this?

Reply:
The point is that basically the smoothing procedure should keep the gradient of DISTANCE constant on all the boundaries (unless the user intentionally wants to keep this requirement only for a specific boundary).

Reply:
OK. I think we should document this somewhere in the code.

Reply:
Done.